src/odemis/gui/util/img.py

```diff
@@ -1037,6 +1037,11 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
     while cur_val < max_val:
         tick_values.append(cur_val)
         cur_val += value_step
+    if len(tick_values) < 2:
+        logging.debug("Only got ticks %s, while wanted at least %d, will fallback to min/max",
+                      tick_values, num_ticks)
```

Comment:
A bit of extra context what these ticks are is nice for the reader of this logging message, this is just a quick example:
  logging.debug("Only got ticks %s, while wanted at least %d, will fallback to min/max **for the ticks on the axis**",
                      tick_values, num_ticks)